db/create-dev-database.ts

```diff
@@ -0,0 +1,26 @@
+import "dotenv/config";
+import { exit } from "process";
+import { pgClientAdminDefaultDB } from "./pg-client";
```

Comment:
How does this create a new DB with `DATABASE_NAME` when it's set to `database: "defaultdb",` in pg-client? Does the env variable `DATABASE_NAME` override it?